app/Controllers/AuthController.js

```diff
@@ -71,7 +71,7 @@ const login = catchAsync(async (req, res) => {
 });
 
 const profile = catchAsync(async (req, res) => {
-  res.send(deleteKeysFromObject(['roleId', 'email', 'password', 'createdAt', 'updatedAt'], req.user.toJSON()));
```

Comment:
In the Frontend you are using the "createdAt" property. You deleted the wrong one

Reply:
No, these changes are correct. We now no longer delete the createdAt from the response.

Reply:
Oh sorry, my bad. I have read over the "deleteKeysFromObject" function